devtools/conda-recipe/meta.yaml

```diff
@@ -15,6 +15,12 @@ requirements:
 
   run:
     - python
+    - numpy
+    - pandas
+    - pytest
+    - jsonschema
+    - requests
+    - pyyaml
```

Comment:
Cool, much lighter than fractal.

Reply:
Yes, but I did have to add pyyaml directly since Fractal gets it from somewhere else